k8s.gcr.io/images/k8s-staging-etcd/OWNERS

```diff
@@ -1,11 +1,19 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
+options:
+  no_parent_owners: true
+approvers:
+  - release-engineering-approvers
+  - cblecker
+  - dims
+  - listx
+  - thockin
 reviewers:
+  - release-engineering-reviewers
   - jpbetz
   - wenjiaswe
   - jingyih
-approvers:
-  - jpbetz
```

Comment:
@justaugustus please add @jpbetz and @jingyih back as approvers. they have been doing the heavy lifting all this time!

Reply:
Discussed in the Slack thread in the PR description.
Root promotion is delegated to Release Managers, so these PRs should not be approved without our review.

Reply:
then please remove everyone except release-engineering-approvers from the approvers list.

Reply:
Done!